docs/source/standard.rst

```diff
@@ -0,0 +1,53 @@
+Card Game Notation Standard
+===========================
+This standard uses TOML_ as notation language.
+
+Header
+-------
+All meta data is defined in the Header.
+This contains information about teams and players.
+Furthermore, the location and datetime is stated.
+
+.. code-block:: toml
+
+    [header]
+    # Timestamp of the starting time in RFC 3339
+    timestamp = 2022-08-06T11:22:00
+    # Location of the game or the server url
+    location = "Hamburg"
+    # Unsigned Integer
+    number_of_teams = 2
+    [[players.player_name_1]]
```

Comment:
it would be easier to not reference the player with a name but with an index
player1, player2, ...

Reply:
then we can also add more metadata like name, nickname, country of origin or something

Reply:
> it would be easier to not reference the player with a name but with an index
> player1, player2, ...

But we loose human readability

Reply:
> then we can also add more metadata like name, nickname, country of origin or something

Sure thing, this is just for the general direction.

Reply:
In toml we can quote parts of dotted identifiers - so weird names are not a problem.
But what about duplicate names?
I would like to see indices to have an unambiguous representation.

Reply:
Fair enough